contracts/SpokePool.sol

```diff
@@ -836,9 +864,9 @@ abstract contract SpokePool is
         // Exclusivity deadline is inclusive and is the latest timestamp that the exclusive relayer has sole right
         // to fill the relay.
         if (
```

Comment:
I have a PR [here](https://github.com/across-protocol/contracts/pull/649) that factors this out to a function. Seems like that will go in, so it might be worth considering that here (it should permit a slight simplification of your change).